docs/AQL/master03-syntax.adoc

```diff
@@ -622,3 +636,42 @@ The `ORDER BY` clause is used to sort the returned results. The keyword `ORDER B
 ORDER BY c/name/value
 --------
 
+=== LIMIT and OFFSET
+
+The `LIMIT` clause and `OFFSET` subclause are used to constrain the result sets generated by the rest of the query.
```

Comment:
I would just put: 'The LIMIT clause is used ...'

Reply:
done